src/Template/Template.js

```diff
@@ -61,5 +83,26 @@ if (typeof HTMLTemplateElement === 'undefined') {
       return el;
     };
 
+    var escapeDataRegExp = /[&\u00A0<>]/g;
+
+    function escapeReplace(c) {
+      switch (c) {
+        case '&':
+          return '&amp;';
+        case '<':
+          return '&lt;';
+        case '>':
+          return '&gt;';
+        case '"':
```

Comment:
I think what happened is it came from ShadowDOM code:
https://github.com/webcomponents/webcomponentsjs/blob/938051272519053867b5bafa7411e7afefdd2c01/src/ShadowDOM/wrappers/HTMLElement.js#L31

they have two regexs because there are two escape modes in the HTML spec, depending on if "attribute mode" is set.
https://html.spec.whatwg.org/multipage/syntax.html#serialising-html-fragments